packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -313,33 +311,52 @@ function buildTroubleshooterData({
   return troubleshooterData;
 }
 
+export type PlatformParams = {
+  platformClientId?: string;
+  platformCancelUrl?: string;
+  platformBookingUrl?: string;
+  platformRescheduleUrl?: string;
+  platformBookingLocation?: string;
+};
+
+export type BookingHandlerInput = {
+  bookingData: Record<string, any>;
+  userId?: number;
+  // These used to come from headers but now we're passing them as params
+  hostname?: string;
+  forcedSlug?: string;
```

Comment:
This are optional to match the req.headers patterns but ideally every place (in app) we call these we need to pass these in. 

However optional makes it a lot easier in tests too because they are not important for 95% of the tests we use when testing `handleNewBooking`